test/mocks/seed.ts

```diff
@@ -322,6 +331,30 @@ async function processMembers({
   return { members: [], memberProfiles: [], items: [] };
 }
 
+async function createItemVisibilities(items: (SeedItem & { path: string })[]) {
+  const visibilities = items.reduce(
+    (acc, { path, isHidden, isPublic }) => {
+      if (isHidden) {
+        acc.push({ item: { path }, type: ItemVisibilityType.Hidden });
+      }
+      if (isPublic) {
+        acc.push({ item: { path }, type: ItemVisibilityType.Public });
+      }
+      return acc;
+    },
+    [] as { item: { path: string }; type: ItemVisibilityType }[],
```

Comment:
To avoid this, you can add this type to the generic on the `reduce`. It should allow you to not have to cast and TD will be happy.